_plugins/lang_path.rb

```diff
@@ -0,0 +1,9 @@
+module Jekyll::CustomFilter
```

Comment:
Should we call this `LangPath` or something?

Reply:
Probably - In reading the docs I thought the namespace had to be that.  Will review. Thx

Reply:
Actually, this plugin is no longer used and can go away.

Reply:
Gone.